dev/tools/update_icons.dart

```diff
@@ -74,7 +74,7 @@ const Map<String, String> _identifierRewrites = <String, String>{
   'class': 'class_',
 };
 
-const Set<String> _mirroredIcons = <String>{
+const Set<String> _mirroredInRTL = <String>{
```

Comment:
nit: is mirrored in rtl icons the same thing as ltr icons. I am not too sure if mirrordinRTL is not the same thing as LTR icons.

Reply:
> What's the plan to get from here to adaptive?

A private class to make Icons.adaptive.X possible, and an update to the script to auto-generated that private class's getters based on icons that include "ios" in the name

Reply:
this means these icons should always match text direction, thus be mirrored vertically when text direction is RTL

Reply:
Suggestion of 'iconsMirroredWhenRTL' if that's clearer.

Reply:
Done